compiler/optimizer/ValuePropagationTable.hpp

```diff
@@ -488,10 +488,10 @@ const ValuePropagationPtr constraintHandlers[] =
 
    constrainIfcmpeq,         // TR::ifacmpeq
    constrainIfcmpne,         // TR::ifacmpne
-   constrainCondBranch,      // TR::ifacmplt
-   constrainCondBranch,      // TR::ifacmpge
-   constrainCondBranch,      // TR::ifacmpgt
-   constrainCondBranch,      // TR::ifacmple
+   constrainIfcmplt,         // TR::ifacmplt
+   constrainIfcmpge,         // TR::ifacmpge
+   constrainIfcmpgt,         // TR::ifacmpgt
+   constrainIfcmple,         // TR::ifacmple
```

Comment:
These handlers look safe for address inequalities, but only because they don't do much with unsigned comparisons